doc/source/changes/version_0_26_1.rst.inc

```diff
@@ -8,3 +8,6 @@ Fixes
 -----
 
 * fixed reading from and writing to Excel sheets with 16384 columns or 1048576 rows (Excel's maximum).
+
+* fixed LArray.split_axes using a custom separator and not using sort=True or when the split labels are
+  ambiguous with labels from other axes (closes :issue:`485`).
```

Comment:
If you have an example for one or both situations, that would be nice. Otherwise, forget it.

Reply:
I have examples for both (see added tests) but I am sceptical adding an example in the changelog for a bug which was not present in the release before adds any value.